component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/MetadataItemDAO.java

```diff
@@ -221,6 +223,23 @@ public List<Tuple> countMetadataItemsByMetadataTypeAndAudienceId(long metadataTy
     }
   }
 
+  public List<Tuple> countMetadataItemsByMetadataTypeAndSpacesIdAndCreatorId(long metadataType,
```

Comment:
Shouldn't return a JPA object to upper layer.

```suggestion
  public List<Long, Long> countMetadataItemsByMetadataTypeAndSpacesIdAndCreatorId(long metadataType,
```